packages/core/src/Stores/Helpers/chart-markers.js

```diff
@@ -88,10 +88,7 @@ const createTickMarkers = contract_info => {
             +_tick.epoch === +contract_info.exit_tick_time ||
             getSpotCount(contract_info, _idx) === contract_info.tick_count;
         const is_exit_spot = isExitSpot(tick, idx);
-        const exit_spot_index = tick_stream.findIndex(isExitSpot);
```

Comment:
According to the new figma design we show marker only for exit spot (for Accumulators), that is whe preexit spot and current last sport were deleted.